webapp-django/crashstats/api/templatetags/jinja_helpers.py

```diff
@@ -1,18 +1,17 @@
+import datetime
 import urllib
 import warnings
-import datetime
-from past.builtins import basestring
-from six import text_type
 
 from django_jinja import library
 import jinja2
+import six
 
 from django.forms.widgets import RadioSelect
 
 
 @library.global_function
 def describe_friendly_type(type_):
-    if type_ is basestring:
```

Comment:
`type_` here is an actual type. It's set to `basestring`, but I claim that's wrong since `basestring` should be used for type checking of a value, not equality checking.

So I changed that instance to `six.text_type` and all the other places where it's checked to be `six.text_type`. That'll be `unicode` in Python 2 and `str` in Python 3. I think that should work fine, but we'll know better when this hits stage and possible weird data.